.github/workflows/test.yml

```diff
@@ -4,7 +4,7 @@ jobs:
   test:
     strategy:
       matrix:
-        rust-toolchain: [1.49.0]
+        rust-toolchain: [1.54.0]
```

Comment:
Should this be changed to `stable`? That's what we've done in [the CLI](https://github.com/fastly/cli/blob/main/.github/workflows/pr_test.yml#L58) for example.